src/lib/sidebar/NoteListItem.js

```diff
@@ -14,6 +15,10 @@ const UnsavedChangesDiv = HorizontalDiv.extend`
   width: 10px;
 `;
 
+const InsetText = styled.div`
+  color: ${props => props.theme.colors.insetText};
+`;
+
```

Comment:
Good work, just add the padding prop from theme also to this! Just like how I did my date in note view.